go/kbnm/install_host

```diff
@@ -16,29 +16,54 @@ realpath() {
   fi
 }
 
```

Comment:
Yea... It was kind of nice to see how the install works as a bunch of commands, but it's pretty redundant with the Go code at this point.